rustls/src/lib.rs

```diff
@@ -312,37 +312,39 @@ mod log {
     macro_rules! error    ( ($($tt:tt)*) => {{}} );
 }
 
+#[macro_use]
+mod check;
+
 #[allow(missing_docs)]
 #[macro_use]
 mod msgs;
+
 mod anchors;
+mod bs_debug;
+mod builder;
 mod cipher;
 mod conn;
 mod error;
 mod hash_hs;
+mod key;
+mod key_log;
+mod key_log_file;
+mod kx;
 mod limited_cache;
 mod rand;
 mod record_layer;
 mod stream;
+mod suites;
+mod ticketer;
```

Comment:
Apparently my IDE decided to automatically alphabetize these. Will deal with that separately.